DESIGN.md

```diff
@@ -117,23 +113,63 @@ ram_reserved = (256, 512, 1024) if total memory (4-7 GB, 8-16 GB, 16 GB+)
 
 > Any Replica should/would/will receive the same settings as the Primary Master, as a Replica is required to have the same system resources as the Primary Master.
 
-#### Large Reference Architecture (Compilers)
+#### Large Reference Architecture (Master and Compilers)
 
-##### PuppetDB Service (pe-puppetdb)
+Calculations for the Master in a Large Ref Arch use the same algorithm as for the Standard Ref Arch with the
+following exceptions:
 
-If this is a Compiler with PuppetDB, the following change ...
+PuppetServer on the Master will process catalog requests only for other PE Infrastructure hosts.
+While PuppetDB on the Master will be expected to handle requests from PuppetServer services on 
+multiple Compilers that together are servicing more agents then the Standard Ref Arch.  So resources
+on the master are transferred from Puppetserver to PuppetDB as follows:
 
 ```
-percent_cpu_puppetdb = 0.25
-minimum_cpu_puppetdb = 1
-maximum_cpu_puppetdb = 3
-```
+percent_cpu_puppetdb = 0.50 #up from 0.25
 
+percent_ram_puppetdb = 0.20 #up from .10
+```
 > In addition, garbage collection is disabled, as it should only be performed by PuppetDB on the Master.
 
-##### PuppetServer Service (pe-puppetserver)
+Compilers are configured by the same algorithm used for the Master in a Standard ref arch. If
+PuppetDB is on the compilers, then that PuppetDB connects to the same PostgresSQL as the
+PuppetDB on the Master.  We restrict PuppetDB's max CPU on the compilers so that PuppetDB on 
+Compilers is limited to a small number of connections which prevents overallocation of 
+connections to PostgresSQL.
+
+```
+maximum_cpu_puppetdb = 3    # was (CPU * 0.50)
+```
+
+#### Extra Large Reference Architecture (Master, Compilers and standalone PuppetDB/Postgres)
```

Comment:
Refering to a standalone PostgreSQL instance in the same breath as a standalone PDB instance can be misread.   This should only refer to an external PDB.  Any assumptions about the location of the PSQL instance in relationship to PDB should be called out explicitly.   I presume this section is designed for a standalone PDB instance on which PSQL is hosted on the same instance.  This should be stated under this title and all references to the host should just read "PuppetDB" without mention of PSQL in order to prevent misinterpreting where the PSQL instance is expected to reside.
```suggestion
#### Extra Large Reference Architecture (Master, Compilers and standalone PuppetDB)
```